src/display/svg.js

```diff
@@ -264,6 +264,7 @@ var SVGExtraState = (function SVGExtraStateClosure() {
     this.dependencies = [];
 
     // Clipping
+    this.pendingClip = null;
```

Comment:
It does not look like you store the pending clip in the `SVGExtraState` object (but instead as a property of `SVGGraphics`), so I think this should be moved to above https://github.com/mozilla/pdf.js/pull/8542/files?w=1#diff-84f753e2bb39669e059ce3447e5a0057R365, just like in `canvas.js`.